stock_lot_filter_available/models/stock_production_lot.py

```diff
@@ -10,9 +12,9 @@ class StockProductionLot(models.Model):
 
     @api.model
     def _is_unsupported_search_operator(self, operator):
-        return operator != ">"
+        return operator not in ["=", "!=", ">", ">=", "<", "<="]
 
-    def _search_product_qty(self, operator, value):
+    def _search_product_qty(self, op, value):
```

Comment:
Is the initial signature with `operator` arg? If so, we shouldn't change it, but do the other thing I explained.